src/View/Widget/DateTimeWidget.php

```diff
@@ -93,6 +94,7 @@ public function __construct(StringTemplate $templates, SelectBoxWidget $selectBo
      * - `second` - Set to true to enable the seconds input. Defaults to false.
      * - `meridian` - Set to true to enable the meridian input. Defaults to false.
      *   The meridian will be enabled automatically if you choose a 12 hour format.
+     * - `timezone` - A hidden field for timezone handling when working with UTC internally.
```

Comment:
Not just UTC really. Any timezone could be stored in the hidden input.

Reply:
The point is that you store anything but UTC usually :) The UTC is the server timezone for DB storing it normalized, and the timezone is either passed in the array from user input (session data?) or coming from config by default (e.g. german based website using global gmt+1 output).
So the idea is UTC => GMT+1 for display => UTC for form handling.
Thus "working with UTC internally" - which seems to be the best practice when dealing with localized datetime across multiple timezones.

Reply:
I agree UTC is what we should recommend, but people will do all sorts of interesting things.